geonode/harvesting/admin.py

```diff
@@ -79,7 +80,7 @@ def save_model(self, request, obj: models.Harvester, form, change):
             available = utils.update_harvester_availability(obj)
             if available:
                 partial_task = functools.partial(
-                    tasks.update_harvestable_resources.apply_async, args=(obj.pk,))
+                    tasks.update_harvestable_num_available_resources.apply_async, args=(obj.pk,))
```

Comment:
with this change we are no longer discovering and saving existing harvestable resources whenever a harvester is created via the admin. This is not the intended behavior.

Please revert this.

Reply:
updated @ricardogsilva